### PR TITLE
ISSUE-46: Show list of sentences

### PR DESCRIPTION
**Pull request by alimuhammadahmer**
This is a pull request containing Ruuter and Resql endpoints for:

1. List of sentences in a corpora (paginated).
2. Showing a particular sentence/task detail.
3. Getting latest corpora info.

### CODE REVIEW

File: DSL.Resql/get_corpora.sql [hunk 1]
```diff
@@ -0,0 +1 @@
+SELECT * FROM list_corpora_tasks_paginated(:where_condition, :sort_condition, :page, :page_size);
```
Comment (turnerrainer) [inline DSL.Resql/get_corpora.sql:1]: Avoid `select *` in any case and be explicit on what you are requesting.

File: DSL.Resql/get_task.sql [hunk 2]
```diff
@@ -0,0 +1 @@
+select *, sentences_annotations::text AS predictions from Corpora_Tasks where corpora_id = :corpora_id and id = :id ORDER BY created_at DESC limit 1;
```
Comment (turnerrainer) [inline DSL.Resql/get_task.sql:1]: Avoid `select *` in any case and be explicit on what you are requesting.

File: README.md [hunk 3]
```diff
@@ -11,5 +11,30 @@ https://github.com/buerokratt/Data-Anonymizer/blob/main/DSL.Resql/insert_regex.s
...
+``` 
+
+### Show list of sentences
+
+https://github.com/buerokratt/Data-Anonymizer/issues/46
+
+https://github.com/buerokratt/Data-Anonymizer/blob/main/DSL.Resql/get_corpora.sql
+
+```
+```
```
Comment (turnerrainer) [inline README.md:24]: Please add a sample curl request.

File: README.md [hunk 4]
```diff
@@ -11,5 +11,30 @@ https://github.com/buerokratt/Data-Anonymizer/blob/main/DSL.Resql/insert_regex.s
...
+
+https://github.com/buerokratt/Data-Anonymizer/blob/main/DSL.Ruuter/GET/tasks.yml
+
 ```
 ```
```
Comment (turnerrainer) [inline README.md:29]: Please add a sample curl request.

File: README.md [hunk 5]
```diff
@@ -11,5 +11,30 @@ https://github.com/buerokratt/Data-Anonymizer/blob/main/DSL.Resql/insert_regex.s
...
+
+https://github.com/buerokratt/Data-Anonymizer/blob/main/DSL.Resql/get_task.sql
+
+```
+```
```
Comment (turnerrainer) [inline README.md:34]: Please add a sample curl request.

File: README.md [hunk 6]
```diff
@@ -11,5 +11,30 @@ https://github.com/buerokratt/Data-Anonymizer/blob/main/DSL.Resql/insert_regex.s
...
+
+https://github.com/buerokratt/Data-Anonymizer/blob/main/DSL.Ruuter/GET/task.yml
+
+```
+```
```
Comment (turnerrainer) [inline README.md:39]: Please add a sample curl request.

**Review by turnerrainer — CHANGES_REQUESTED** [review]

File: DSL.Resql/get_corpora.sql (hunk 1 above)
Comment (alimuhammadahmer) [reply DSL.Resql/get_corpora.sql]: Done

File: DSL.Resql/get_task.sql (hunk 2 above)
Comment (alimuhammadahmer) [reply DSL.Resql/get_task.sql]: Done

File: README.md (hunk 3 above)
Comment (alimuhammadahmer) [reply README.md]: Done

File: README.md (hunk 4 above)
Comment (alimuhammadahmer) [reply README.md]: Done

File: README.md (hunk 5 above)
Comment (alimuhammadahmer) [reply README.md]: Done

File: DSL.Resql/get_corpora.sql [hunk 7]
```diff
@@ -0,0 +1 @@
+SELECT id, corpora_id, created_at, full_count, is_private, predictions, raw_text FROM list_corpora_tasks_paginated(:where_condition, :sort_condition, :page, :page_size);
```
Comment (turnerrainer) [inline DSL.Resql/get_corpora.sql:1]: No newline at end of file

File: DSL.Resql/get_task.sql [hunk 8]
```diff
@@ -0,0 +1 @@
+select id, corpora_id, created_at, is_private, raw_text, sentences_annotations::text AS predictions from Corpora_Tasks where corpora_id = :corpora_id and id = :id ORDER BY created_at DESC limit 1;
```
Comment (turnerrainer) [inline DSL.Resql/get_task.sql:1]: No newline at end of file

**Review by turnerrainer — CHANGES_REQUESTED** [review]
No newline at end of 2 files, otherwise OK

File: DSL.Resql/get_corpora.sql (hunk 7 above)
Comment (alimuhammadahmer) [reply DSL.Resql/get_corpora.sql]: Done

File: DSL.Resql/get_task.sql (hunk 8 above)
Comment (alimuhammadahmer) [reply DSL.Resql/get_task.sql]: Done

**Review by turnerrainer — APPROVED** [review]

**Pull request merged**